packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts

```diff
@@ -102,6 +115,32 @@ describe('scaffolding component testing', {
       cy.get(`[data-testid="select-framework"]`)
 
       cy.get('button').should('be.visible').contains('React.js(detected)')
+
+      cy.get('button').contains('Next step').click()
+
+      // react-dom dependency is missing
+      cy.findByTestId('dependency-react-dom').within(() => {
+        cy.get('[aria-label="pending installation"]').should('exist')
+      })
+
+      // fake install
+      cy.withCtx(async (ctx) => {
+        await ctx.fs.mkdirp(ctx.path.join(ctx.currentProject!, 'node_modules', 'react-dom'))
+        await ctx.actions.file.writeFileInProject(
+          ctx.path.join('node_modules', 'react-dom', 'package.json'),
+          JSON.stringify({
+            'version': '17.0.0',
+          }),
+        )
+      })
+
+      // Wait for new dependency detection polling
+      cy.wait(3000)
```

Comment:
Is the explicit `wait` necessary here? Could we bump up the timeout on the following command to compensate?

Reply:
I'll try removing it, it shouldn't be necessary.